presidio-analyzer/tests/test_analyzer_engine.py

```diff
@@ -273,7 +273,7 @@ def test_when_entities_is_none_all_recognizers_loaded_then_return_all_fields(
         registry=RecognizerRegistry(), nlp_engine=nlp_engine
     )
     threshold = 0
-    text = "My name is David and I live in Seattle." "Domain: microsoft.com "
+    text = "My name is Sharon and I live in Seattle." "Domain: microsoft.com "
```

Comment:
😆

Reply:
For some reason David wasn't recognized. *I plan to stay on the Israeli site

Reply:
Well, David is a very basic name. that is very weird. I would add @omri374 to this conversation.

Reply:
:) https://github.com/explosion/spaCy/issues/7493#issuecomment-803459979

Reply:
@omri374 and we still want to upgrade?